js/App.js

```diff
@@ -1,38 +1,64 @@
 import React, { Component } from 'react';
-import { Platform, StyleSheet, Text, View } from 'react-native';
-
-const instructions = Platform.select({
-  ios: 'Press Cmd+R to reload,\n' + 'Cmd+D or shake for dev menu',
-  android:
-    'Double tap R on your keyboard to reload,\n' +
-    'Shake or press menu button for dev menu',
-});
+import styles from './styles';
+import { View, Text, Image, Alert } from 'react-native';
+import Splash from './components/Splash/Splash';
 
 export default class App extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      isVisible: true,
+    };
+  }
+
+  hide_splash = () => {
+    this.setState({
+      isVisible: false,
+    });
+  };
+
+  componentDidMount() {
+    var that = this;
```

Comment:
This is an anti-pattern. Please use an arrow function in your setTimeout and avoid using the var keyword.